src/demo/demoUtils.js

```diff
@@ -26,6 +26,7 @@ export const getInitSdkOptions = () => {
   const steps = [
     'welcome',
     queryParamToValueString.poa === 'true' && { type: 'poa' },
+    !queryParamToValueString.faceDetection === 'true' &&
```

Comment:
Please remove this line so that the tests pass and TravisCI will generate a preview link.